guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -5,34 +5,36 @@ For {SmartProxies} configured to use `pull-mqtt` mode, hosts can subscribe to re
 Managed hosts do not require an SSH connection to their {SmartProxyServer}.
 
 .Prerequisites
-* You have registered the host to {ProjectName}.
+* You have registered the host to {Project}.
 * The host's {SmartProxy} is configured to use `pull-mqtt` mode.
 * The {Team} {project-client-name} repository is enabled and synchronized on {ProjectServer}, and enabled on the host.
 * The host is able to communicate with its {SmartProxy} over MQTT using port `1883`.
 * The host is able to communicate with its {SmartProxy} over HTTPS.
 
+NOTE: The `katello-pull-transport-migrate` package was created to help users migrate from `katello-agent` to remote execution with the pull client.
+However, having `katello-agent` installed on the host is not a requirement.
+You can use `katello-pull-transport-migrate` regardless of whether `katello-agent` is installed.
+
 .Procedure
 . Install the `katello-pull-transport-migrate` package on your host:
-
-On EL7 hosts:
+** On EL7 hosts:
++
 ----
 # yum install katello-pull-transport-migrate
 ----
-
-On EL8 and EL9 hosts:
+** On EL8 and EL9 hosts:
++
 ----
 # dnf install katello-pull-transport-migrate
 ----
```

Comment:
I believe hosts should be mentioned in reversed order here: first EL 9 and EL 8 hosts section, then EL 7 hosts section.